types/strophe.js/tslint.json

```diff
@@ -0,0 +1,21 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "comment-format": false,
```

Comment:
Sorry to keep making you go through this, but we'd really like to get all of these rules enabled, since most indicate some issue you'll hit in the future. Can you document where each of these is causing issues? I'll be able to help resolve them then.

Reply:
Especially `dt-header`, that one is exceptionally important.